web/styles/breakpoints.sass

```diff
@@ -152,3 +161,15 @@ $mobile-s:  320px
             @media screen and (max-width: $full-screen)
               cursor: default !important
               opacity: 100 !important
+      &-desktop
+        &:hover
+          @media (hover:hover)
+            @media screen and (max-width: $desktop)
+              cursor: default !important
+              opacity: 100 !important
+      &-stories
+        &:hover
```

Comment:
Ooo nice, I think putting `&:hover` before the `@media (hover:hover)` avoids the Firefox hover bug.